app/src/qvm-app-version.lisp

```diff
@@ -48,18 +48,21 @@
             version)))
     (usocket:ns-error (condition)
       (declare (ignore condition))
-      (cl-syslog:rfc-log (*logger* :warning "Encountered a name resolution error when fetching latest SDK version. (~A)" condition)
-        (:msgid "LOG0001"))
+      (with-locked-log ()
```

Comment:
Using `with-locked-log` here might be overkill. This function only appears to be called at startup, before any other threads have been created.